modules/clinical/conservationMethod.csv

```diff
@@ -0,0 +1,7 @@
+Attribute,Description,Valid Values,DependsOn,Properties,Required,Parent,DependsOn Component,Source,Validation Rules,columnType,module
+conservationMethod,The conservation method,"None, fixed tissues only, frozen tissue only, both fixed and frozen tissue, missing or unknown",,,False,ManifestColumn,,sage-annotations-clinical.conservationMethod-0.0.2,string,clinical
```

Comment:
```suggestion
conservationMethod,The conservation method,"None, fixed tissues only, frozen tissue only, both fixed and frozen tissue, missing or unknown",,,False,ManifestColumn,,sage-annotations-clinical.conservationMethod-0.0.2,,string,clinical
```

Reply:
GAHHH found an error using the Rainbow CSV Lint function in VS code -- missed an empty column so this row only had eleven columns instead of twelve. See screenshot that helped me figure it out

<img width="576" alt="Screenshot 2024-05-09 at 11 11 29 AM" src="https://github.com/adknowledgeportal/data-models/assets/11965371/6c0fcee0-18e9-4c18-b093-6ddfe81d1e59">